src/Orchard.Web/Modules/Orchard.OpenId/Constants.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Orchard.OpenId {
+    public class Constants {
+        public const string AuthenticationErrorUrl = "~/Authentication/Error";
+        public const string LogonCallbackUrl = "~/Users/Account/LogonCallback";
+        public const string OpenIdOwinMiddlewarePriority = "10";
+        public const string LocalIssuer = "LOCAL AUTHORITY";
+        public const string FormsIssuer = "Forms";
+
+        public const string AzureActiveDirectoryObjectIdentifierKey = "http://schemas.microsoft.com/identity/claims/objectidentifier";
+
+        // Certificate Subject Key Identifier
+        public const string VeriSignClass3SecureServerCA_G2 = "A5EF0B11CEC04103A34A659048B21CE0572D7D47";
```

Comment:
Is there the same issue for Azure AD, I can remember they regularly recycle some keys or thumbprints.